3p/ampcontext-integration.js

```diff
@@ -69,7 +69,8 @@ export class IntegrationAmpContext extends AbstractAmpContext {
     // available.
     return (this.embedType_ === 'facebook'
         || this.embedType_ === 'twitter'
-        || this.embedType_ == 'github');
+        || this.embedType_ == 'github'
```

Comment:
as you mentioned, let's change the `==` to be consistent.

Reply:
right, fixed in https://github.com/ampproject/amphtml/pull/12836/commits/d1bfee5726bed9c061e5e1c1151bc247d8488464